data/software.json

```diff
@@ -1954,10 +1954,19 @@
             "tool"
         ],
         "doap": "/hosted-doap/xmpp-account-exporter.doap",
-        "name": "XMPP account exporter",
+        "name": "XMPP Account Exporter",
```

Comment:
This has to be the same name as the one listed in the tool's DOAP file. Both the DOAP file _and_ the website use lower case writing ;)